neotest_python/pytest.py

```diff
@@ -4,7 +4,10 @@
 
 from .base import NeotestAdapter, NeotestError, NeotestResult, NeotestResultStatus
 
+import pytest
```

Comment:
This introduces a module-level runtime dependency on `pytest`. Given that `.pytest` is dynamically loaded only when the runner is "pytest" (and `.run()` is called soon thereafter, which performs the import too), I think this is safe to do?

Reply:
Correct, this is fine :+1: